README.md

```diff
@@ -6,7 +6,11 @@
 
 # Readme
 
-Explain your project here.
+Contact Book is a file system for storing personal contact details.
+
+## System Diagram
+![](diagram.png)
```

Comment:
The square brackets in markdown images are used for what in HTML would be the `alt` attribute of an image, where you usually add some text describing the image for people with screen readers.

Although it is not mandatory on a README it's always nice to add it because another good side effect is that if the image is not available or can't be downloaded for some reason, the text will be displayed instead.

You could also add a small paragraph in this section explaining that "you can keep the diagram up to date uploading `diagram.xml` to https://draw.io" (I believe, check it out just in case)